docs/resolve/relationships.rst

```diff
@@ -142,50 +142,98 @@ This is by design because you wouldn't want one component to dispose the ``B`` o
+      public void DoSomething() {}
+    }
 
     public class A
     {
-        Func<int, string, B> _b;
+        Func<int, string, B> _newB;
 
-        public A(Func<int, string, B> b) { _b = b }
+        public A(Func<int, string, B> b) { _newB = b }
 
         public void M()
         {
-            var b = _b(42, "http://hel.owr.ld");
+            var b = _newB(42, "http://hell.owor.ld");
+            b.DoSomething();
+        }
+    }
+
+Note that since we're ``.Resolve()``ing the instantiation, rather than actually directly calling the constructor we don't need to declare the parameters in the same order as they appear in the ctor, nor do we necessarily need to provide *all* the parameters listed in that constructor. If some of the constructor's parameters could have been resolved by the Autofac Container already, then those parameters can be omitted from the ``Func`` signature being declared, so you only need to list the types that the Container can't resolve.
+
+Alternatively, you can use this approach to override a constructor parameter that *would* otherwise have been resolved from the container, with a concrete instance already in hand.
+
+Example:
+.. sourcecode:: csharp
+    //Suppose that P, Q & R are all registered with the Autofac Container.
+    public class B
+    {
+      public B(int id, P peaDependency, Q queueDependency, R ourDependency) {}
+      
+      public void DoSomething() {}
+    }
+
+    public class A
+    {
+        Func<int, P, B> _newB;
+
+        public A(Func<int, P, B> bFactory) { _newB = bFactory }
+
+        public void M(P existingPea)
+        {
+            var b = _newB(42, existingPea); //the needed Q and R will be resolved by Autofac, but not the P since the existingPea will be used.
             b.DoSomething();
         }
     }
 
-Internally, Autofac treats these as typed parameters. What that means is that **auto-generated function factories cannot have duplicate types in the input parameter list.** For example, say you have a type like this:
+Internally, Autofac determines what values to use for the constructor args, solely based on the types, and behaves as though we've temporarily defined the input values as the objects that should be used to resolve those types. A direct consequence of this is that  **auto-generated function factories cannot have duplicate types in the input parameter list.** See below for further notes on this.
+
+
+**Lifetime scopes are respected** using this relationship type, just as they are when using delegate factories. If you register an object as ``InstancePerDependency()`` and call the ``Func<X, Y, B>`` multiple times, you'll get a new instance each time. However, if you register an object as ``SingleInstance()`` and call the ``Func<X, Y, B>`` to resolve the object more than once, you will get *the same object instance every time regardless of the different parameters you pass in.* Just passing different parameters will not break the respect for the lifetime scope.
```

Comment:
By moving this further away from the discussion about delegate factories, the "just as they are..." part gets disconnected. We need to link to add that context back.

```suggestion
**Lifetime scopes are respected** using this relationship type, just as they are when using :doc:`delegate factories <../advanced/delegate-factories>`. If you register an object as ``InstancePerDependency()`` and call the ``Func<X, Y, B>`` multiple times, you'll get a new instance each time. However, if you register an object as ``SingleInstance()`` and call the ``Func<X, Y, B>`` to resolve the object more than once, you will get *the same object instance every time regardless of the different parameters you pass in.* Just passing different parameters will not break the respect for the lifetime scope.
```